pkg/workflows/sdk/builder.go

```diff
@@ -238,3 +249,23 @@ func AnyMap[M ~map[string]any](inputs CapMap) CapDefinition[M] {
 
 	return components
 }
+
+type Secret string
+
+func (s Secret) Ref() any {
+	return s
+}
+
+func (s Secret) private() {}
+
+func (s Secret) self() CapDefinition[string] {
+	return s
+}
+
+func Secrets() Secret {
+	return "$(ENV.secrets)"
+}
+
+func SecretNamed(named string) Secret {
```

Comment:
nit: I think Secret(name sting) is more natural.  The arg implies what you're doing, and it's clear that one gets all secrets and the other gets on secret.

Reply:
This is what I had initially but then I don't know what to call the Secret type; maybe SecretValue?

Reply:
I've called the Secret type SecretValue and renamed this to Secret